clang/lib/DPCT/SrcAPI/APINames_cuTENSOR.inc

```diff
@@ -0,0 +1,74 @@
+*    default(all bits are 0): NO_FLAG
+*    bit 0: API_CALL_REMOVED
+*    bit 1: API_CALL_UNSUPPORTED
+*  Priority: migration priority, eg. P0 - P4
+*    P0:  should support in Beta release.
+*    P4:  should support in Gold.
+*
+*  MigrationDesc: Comment[: Comment] | Unsupported: Comment
+*    Comment:   this API has been migrated.
+*    Unsupported:  this API can not be migrated currently.
+*                  Must add Comment to explain why this API is Unsupported.
+*    Comment:      extra comment on the migration.
+*
+****************************************************************************/
+
+// Helper Functions
+ENTRY(cutensorCreate, cutensorCreate, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorDestroy, cutensorDestroy, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorCreateTensorDescriptor, cutensorCreateTensorDescriptor, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorDestroyTensorDescriptor, cutensorDestroyTensorDescriptor, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorOperationDescriptorGetAttribute, cutensorOperationDescriptorGetAttribute, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorOperationDescriptorSetAttribute, cutensorOperationDescriptorSetAttribute, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorDestroyOperationDescriptor, cutensorDestroyOperationDescriptor, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorCreatePlanPreference, cutensorCreatePlanPreference, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorDestroyPlanPreference, cutensorDestroyPlanPreference, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorPlanPreferenceSetAttribute, cutensorPlanPreferenceSetAttribute, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorEstimateWorkspaceSize, cutensorEstimateWorkspaceSize, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorCreatePlan, cutensorCreatePlan, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorDestroyPlan, cutensorDestroyPlan, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorGetErrorString, cutensorGetErrorString, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorGetVersion, cutensorGetVersion, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorGetCudartVersion, cutensorGetCudartVersion, false, NO_FLAG, P4, "comment")
+
+
+// Element-wise Operations
+ENTRY(cutensorCreatePermutation, cutensorCreatePermutation, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorPermute, cutensorPermute, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorCreateElementwiseBinary, cutensorCreateElementwiseBinary, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorElementwiseBinaryExecute, cutensorElementwiseBinaryExecute, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorCreateElementwiseTrinary, cutensorCreateElementwiseTrinary, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorElementwiseTrinaryExecute, cutensorElementwiseTrinaryExecute, false, NO_FLAG, P4, "comment")
+
+
+// Reduction Operations
+ENTRY(cutensorCreateReduction, cutensorCreateReduction, false, NO_FLAG, P4, "comment")
+ENTRY(cutensorReduce, cutensorReduce, false, NO_FLAG, P4, "comment")
```

Comment:
cutensorCreateContractionTrinary is missing

Reply:
cutensorLoggerForceDisable is missing

Reply:
Updated the API list now